### DISCUSSION TITLE
Images in readme.md are randomly breaking in some assignment repositories.

### DISCUSSION THREAD

**Discussion by EmeryAtRTC** [community/community#68547] [GitHub Education]
### Hi everyone,

I have been using GitHub classrooms for assignments without issue for about a year now. Recently I have started providing more images in the readme markdown file to my students to better guide them through the process. I typically will copy paste the image directly into the markdown file. This results in a link that looks something like this:
` ![image](https://github.com/AccountName/RepoName/assets/102991550/f245b2da-4f49-4ee0-a4d6-268367f9e0f8)`

I understand that it would be possible to upload the images to the repo and then link to them using HTML, however this would significantly increase the amount of time it takes to create the assignments. Some of them have 20 or more screenshots giving students directions.

These seem to work about 90% of the time, but occasionally when a student clones an assignment repository all of the images will be broken. When referring to my template repository the images still work. I also have had one instance where this happened in my template repository. I recaptured all of the images and created the readme.md again. Any idea why this is happening?

Thanks

**Comment by mantasu** [comment]
Try using `raw` version of the image, for example, `https://raw.githubusercontent.com/AccountName/RepoName/BranchName/assets/my_image.png`

**Comment by smileBeda** [comment]
This is SOMEHOW an issue with _the CDN GitHub uses_. I do not understand WHY, but, for example over a VPN, things work, while on standard connection (suddenly) most readme images (not only my own projects) go with missing images (even if they are not missing).
The error is a "too many requests"  429 error in the console.

The problem is to 99% rate limiting or CDN-level throttling based on our IP addresses, and this here is NOT the only report of this issue to GitHub!

**Comment by mayurcodes01** [comment]


**Reply by smileBeda** [reply]
Sorry but keep the AI slob on your own desk.

**Comment by roshanthm** [comment]


**Reply by smileBeda** [reply]
Exactly wrong. Did you even read the description and replies to this topic!?

**Comment by mayurcodes01** [comment]


**Reply by smileBeda** [reply]
Contribution farming much?
I have reported yours and the others account. Please refrain from abusing GitHub issues for contribution farming.
